CHANGES.rst

```diff
@@ -4,15 +4,14 @@ Changelog
 2.2+md.4 (unreleased)
 =====================
 
-- Nothing changed yet.
-
+- Add "eggs" working set to template context when it is mentioned in the part.
+  Shamelessly copied from `gobre.recipe.template`.
```

Comment:
An example would be nice how to use it.

Fun fact: "gobre" stands for "gocept buildout recipe". We dit not migrate it to MD because it was used in union.cms, so we moved the repository to the union.cms maintainers.